headers.h

```diff
@@ -11,15 +11,30 @@
 #define HAVE_SETSID
 
 /* OS-specific includes */
-#if defined(__GLIBC__)
+#if defined(__GLIBC__) || defined(__CYGWIN__)
+# define HAVE_UNISTD_H
+# define HAVE_SYS_IOCTL_H
```

Comment:
This breaks compilation with musl libc. According to the main code, the include should be conditional on `__linux__` instead of `__GLIBC__`.